.github/workflows/pypi.yml

```diff
@@ -11,13 +11,10 @@ on:
 
 # Cancels all previous workflow runs for pull requests that have not completed.
 concurrency:
-  # The concurrency group contains the workflow name and the branch name for pull requests
-  # or the commit hash for any other events.
```

Comment:
This comment seems not useless, any particular reason to delete it?

Reply:
That was GPT, I agree.